backend/src/agents/crew.py

```
@@ -0,0 +1,88 @@
+@CrewBase
+class PlannerCrew:
+    """Planner Crew"""
+
+    def __init__(self):
+        # Load YAML files into dictionaries
+        with open("config/agents.yaml", 'r') as file:
+            self.agents_config = yaml.safe_load(file)
+        
+        with open("config/tasks.yaml", 'r') as file:
+            self.tasks_config = yaml.safe_load(file)
+    print(self.agents_config)
```

Ruff (0.12.2)

Undefined name `self`

(F821)